salt/loader.py

```diff
@@ -1062,7 +1064,8 @@ def __init__(self,
         # names of modules that we don't have (errors, __virtual__, etc.)
         self.missing_modules = {}  # mapping of name -> error
         self.loaded_modules = {}  # mapping of module_name -> dict_of_functions
-        self.loaded_files = set()  # TODO: just remove them from file_mapping?
+        self.loaded_files = {}  # TODO: just remove them from file_mapping?
```

Comment:
It's now mapping file names to the module objects, so set/array isn't enough and dict is used.